frontend/src/components/Modal.svelte

```diff
@@ -57,15 +64,14 @@
         background-color: var(--background);
         padding: 0.2rem;
         border-radius: 0.6rem;
-        min-width: 20vw;
```

Comment:
TODO: add these back

Reply:
why? the min-width/height is variable now, and defaults to 20. I did this so they can be set when using the component

Reply:
min width/height shouldn't be a prop and belongs in the child components imo

Reply:
+ modals smaller than these are weird and these are good min values